hello.py

```diff
@@ -870,22 +892,41 @@ def main():
         registry2=registry2,
         snapshotter=snapshotter,
         cleanup=cleanup,
+        insecure_registry=insecure_registry,
     )
 
+    if output_format == "csv":
+        csv_headers = (
+            "timestamp,repo,bench,pull_time(s),create_time(s),run_time(s),elapsed(s)"
```

Comment:
pull_times(s) looks strange, is `time` countable?

Reply:
Yes, I have replaced it with `elapsed`.